generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.ts.ejs

```diff
@@ -26,7 +26,7 @@ import { ActivatedRoute } from '@angular/router';
 import { Subscription } from 'rxjs';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import { filter, map } from 'rxjs/operators';
-import { JhiEventManager, <% if (pagination !== 'no') { %>JhiParseLinks, <% } %>JhiAlertService<% if (fieldsContainBlob) { %>, JhiDataUtils<% } %> } from 'ng-jhipster';
+import { JhiEventManager, <% if (pagination !== 'no') { %>JhiParseLinks, <% } %><% if (fieldsContainBlob) { %>, JhiDataUtils<% } %> } from 'ng-jhipster';
```

Comment:
Test failed because of double comma, I think that:
* comma after `JhiEventManager` should be moved before `JhiParseLinks`
* and comma after `JhiParseLinks` should be deleted